src/univariate/continuous/normal.jl

```diff
@@ -150,8 +150,8 @@ function suffstats(::Type{<:Normal}, x::AbstractArray{T}, w::AbstractArray{Float
     n = length(x)
 
     # compute s
-    tw = 0
-    s = 0
+    tw = zero(T)
```

Comment:
This should just be
```suggestion
    tw = 0.0
```
since `w isa AbstractVector{Float64}`.